flake8 >= 3.8.0 causes "F541 f-string is missing placeholders" error during style checking.

**Describe the bug**

flake8==3.8.1 (and 3.8.0) has been released on 5/11.
https://pypi.org/project/flake8/#history

An issue has been reported (https://gitlab.com/pycqa/flake8/-/issues/648), but this seems to be the expected behavior of flake8 >= 3.8.0.
> As of flake8 >= 3.8.0, pyflakes got bumped to >= 2.2.0, which has added a new check that will flag F541.
https://gitlab.com/pycqa/flake8/-/issues/648

**To Reproduce**
Steps to reproduce the behavior:
1. Upgrade flake8 to 3.8.X
2. Execute `make check`

**Expected behavior**

Check runs without an error.

**Error Logs/Screenshots**

```
$ make check
isort -rc -c src/
Skipped 1 files
isort -rc -c tests/
black src/ --check
All done! ✨ 🍰 ✨
88 files would be left unchanged.
black tests/ --check
All done! ✨ 🍰 ✨
27 files would be left unchanged.
flake8 src/
src/fonduer/utils/utils_parser.py:24:24: F541 f-string is missing placeholders
1     F541 f-string is missing placeholders
1
make: *** [check] Error 1
```

**Environment (please complete the following information):**
 - Fonduer Version: latest commit on the master branch (8e321ce8518090e3dd7e863e204f9fad0988a1be).

**Additional context**
Add any other context about the problem here.

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -1,5 +1,5 @@
 black>=18.9b0
-flake8
+flake8>=3.8.0
 mypy
 isort
 pre-commit

--- a/src/fonduer/utils/utils_parser.py
+++ b/src/fonduer/utils/utils_parser.py
@@ -21,7 +21,7 @@ def build_node(type: str, name: str, content: str) -> str:
         return f"<p name='{name}'>{content}</p>"
     if type == "figure":
         return f"<img name='{name}' src='{content}'/>"
-    raise RuntimeError(f"unknown type")
+    raise RuntimeError("unknown type")
 
 
 def column_constructor(

--- a/tests/candidates/test_candidates.py
+++ b/tests/candidates/test_candidates.py
@@ -528,12 +528,12 @@ def test_pickle_subclasses():
     Temp = mention_subclass("Temp")
     PartTemp = candidate_subclass("PartTemp", [Part, Temp])
 
-    logger.info(f"Test if mention/candidate subclasses are picklable")
+    logger.info("Test if mention/candidate subclasses are picklable")
     pickle.loads(pickle.dumps(Part))
     pickle.loads(pickle.dumps(Temp))
     pickle.loads(pickle.dumps(PartTemp))
 
-    logger.info(f"Test if their objects are pickable")
+    logger.info("Test if their objects are pickable")
     part = Part()
     temp = Temp()
     parttemp = PartTemp()

